qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/AllergiesSteps.java

```diff
@@ -17,46 +17,63 @@
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
 
 import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.AddOrEditAllergyPage;
 import org.openmrs.contrib.qaframework.page.AllergyPage;
+import org.openmrs.contrib.qaframework.page.FindPatientPage;
```

Comment:
do you really need the import package for `FindPatientPage`?

Reply:
> do you need the import package for `FindPatientPage`?

@kdaud  how else would line 43 (`findPatientPage = (FindPatientPage) homePage.goToFindPatientRecord().waitForPage();`) run if FindPatientPage was not imported? Any ideas?

Reply:
One way is to inherit it from step definition class. you simply have to call it, However still this looks fine to get moving.

Reply:
@Irenyak1 do you find this documentation helpful? https://www.geeksforgeeks.org/inheritance-in-java/